apps/rotm/translations/src/en/validation.json

```diff
@@ -0,0 +1,10 @@
+{
+  "image": {
+    "required": "Select an image",
+    "extname": "Select a valid file type (jpg, jpeg, png)",
+    "unsaved": "We have not been able to upload the photo or screenshot"
+  },
+  "add-image": {
+    "required": "Tell us if you want to add this image"
+  }
+}
```

Comment:
Can I petition to put validation messages in with the rest of the field messaging - see https://github.com/UKHomeOffice/rotm/blob/master/apps/rotm/translations/src/en/fields.json#L5-L7

It means that all the content related to a field is in one place, and not spread across multiple files.

Reply:
not a bad shout